### DISCUSSION TITLE
post/powerbi-date-table-anniversary-milestone/

### DISCUSSION THREAD

**Discussion by giscus** [AlexanderHenkel/blogComments#4] [Announcements]
# post/powerbi-date-table-anniversary-milestone/

Learn how to calculate age, anniversaries and milestone birthdays based on date of birth and date of employment

https://alexanderhenkel.dk/post/powerbi-date-table-anniversary-milestone/

**Comment by Neeljy21** [comment]
Great post! A well-structured date table is essential for leveraging time intelligence in Power BI effectively. Your explanation of generating a proper date table really helps clarify how to enable dynamic date-based calculations.

**Reply by AlexanderHenkel** [reply]
Thank you for your kind feedback 🚀 as you write, a date table does take out a lot of pain when working in Power BI
